spec/tasks/one_off_spec.rb

```diff
@@ -29,7 +29,7 @@
     end
   end
 
-  describe "backfill kind of Pypi dependencies" do
+  xdescribe "backfill kind of Pypi dependencies" do
```

Comment:
are we intending to skip this?

Reply:
yeah, I left it in the code because it's useful to have to make any tweaks to the rake task, but I think it's wasteful to run it as part of the test suite on every commit into main.